src/App.js

```diff
@@ -1,12 +1,33 @@
 import "./App.css";
 import Header from "./Components/Header";
 import Categories from "./Components/Categories";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import ReviewPageBody from "./Components/ReviewPageBody";
+import { useState } from "react";
 
 const App = () => {
+  const [reviewCategory, setReviewCategory] = useState("");
+
   return (
     <div>
       <Header />
-      <Categories />
+      <BrowserRouter>
+        <Routes>
+          <Route
+            path="/home"
```

Comment:
I strongly encourage you to use '/' as the default home route for your app

Reply:
will do